### PR TITLE
Fixed an issue with the download button not working correctly.

### PR DESCRIPTION
**Pull request by TheOnlyTails**
## Description
This PR extends #64 by fixing an issue with the download button, causing it to use the wrong URL for the sideload package sometimes.

### CODE REVIEW

File: src/layout/HeroSection/HeroSection.svelte [hunk 1]
```diff
@@ -41,34 +34,24 @@
...
-				break;
-			case "Microsoft Store":
-			case "Sideload Package":
-			case "Sideload Package (Preview)":
-				window.open(downloadSource === "Microsoft Store" ? getStoreUrl() : sideloadLink, "_blank");
-				break;
-		}
+		if (downloadSource !== "Winget (CLI)")
+			window.open(downloadSource === "Microsoft Store" ? getStoreUrl() : sideloadLink, "_blank");
+		else wingetDialogOpen = true;
```
Comment (chingucoding) [inline src/layout/HeroSection/HeroSection.svelte:44]: nit: Use curly braces for if and else bodies.
Not nit: Don't put the else body in the same line as the else keyword.

File: src/layout/HeroSection/HeroSection.svelte [hunk 2]
```diff
@@ -41,34 +34,24 @@
...
 		isDownloadDropdownOpen = false;
 	};
 
 	onMount(async () => {
 		// Get the user's download preference
-		if (!localStorage.getItem("downloadSource"))
-			localStorage.setItem("downloadSource", "0");
-		currentDownloadSource =
-			parseInt(localStorage.getItem("downloadSource")) ?? 0;
+		if (!localStorage.getItem("downloadSource")) localStorage.setItem("downloadSource", "0");
```
Comment (chingucoding) [inline src/layout/HeroSection/HeroSection.svelte:51]: Same as above, prefer curly braces. Please don't put the if keyword and the body on the same line.

**Review by chingucoding — APPROVED** [review]
Looks good to me, thank you for fixing this!

**Comment by chingucoding** [comment]
@yaichenbaum Do you also want to review this or can I merge?

**Review by yaichenbaum — APPROVED** [review]

**Pull request merged**